k8stypes/resources.go

```diff
@@ -125,3 +125,14 @@ func NewCronJob() *CronJobV1Beta1 {
 	cronJob.APIVersion = "batch/v1beta1"
 	return cronJob
 }
+
+// NewServiceAccount creates a new ServiceAccount resource
+func NewServiceAccount() *ServiceAccountV1 {
+	serviceAccount := &ServiceAccountV1{
+		ObjectMeta: ObjectMetaV1{},
+	}
+
+	serviceAccount.Kind = "ServiceAccount"
+	serviceAccount.APIVersion = "v1"
```

Comment:
Is there a difference between setting the `Kind` and `APIVersion` this way vs using the `SetGroupVersionKind` like the `GetServiceAccounts` function?

```
serviceAccount.SetGroupVersionKind(schema.GroupVersionKind{
	Kind:    "ServiceAccount",
	Version: "v1",
})
```

Reply:
I believe they are effectively the same. `SetGroupVersionKind` is implemented on the interface so it can be called on both the generic `k8stypes.Resource` that we pass around in most of kubeaudit and on the specific resource types. Good catch though, I'll clean up the types so they're more consistent.

Reply:
Sounds good! 👍